assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -132,11 +131,8 @@ export default function UserInputQuestionnaire() {
 			return;
 		}
 
-		// The `activeSlugIndex` deals with a zero-based index, but the IDs
-		// we need to select use a one-based index, hence the ` + 1` here for
-		// the `activeSlugIndex` selector.
-		global.document.getElementById( `googlesitekit-user-input-question-${ activeSlugIndex + 1 }` )?.scrollIntoView( { behavior: 'smooth' } );
-	}, [ activeSlugIndex ] );
+		global.document.querySelector( '#googlesitekit-user-input-container' )?.scrollIntoView( { behavior: 'smooth' } );
```

Comment:
@tofumatt I believe it would be better to use a ref for the container element instead of relying on its ID. Could you please update it?